src/components/textbox/textbox.component.tsx

```diff
@@ -237,8 +237,6 @@ export const Textbox = ({
             validationRedesignOptIn ? undefined : validationIconId
           }
           validationRedesignOptIn={validationRedesignOptIn}
-          size={size}
-          readOnly={readOnly}
```

Comment:
looking at the internals of `FormField`, while it doesn't actually use either of these props, there is an invariant message telling people to use `readOnly` if trying to use `disabled` with validation props. We should update that (although I'm not sure what to!) if `readOnly` isn't actually accepted. (We should probably just add `readOnly` as a new prop that gets passed to the HTML element.)

Reply:
I think leaving the invariant in `FormField` is correct as it will mean all inputs will have this guard and the prop isn't used by `FormField` so it not being passed isn't and issue. I've also confirmed `readOnly` is passed to the input element.